components/utils/parameters/pagination.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+import logging
+
+class PaginationTraitsBase():
```

Comment:
So we should be able to create a class which wraps PaginationTraitsBase and implement iterator/iterable